src/engine/zsm.cpp

```diff
@@ -278,6 +281,11 @@ void DivZSM::flushWrites() {
   flushTicks(); // only flush ticks if there are writes pending.
   for (unsigned char i=0; i<64; i++) {
     if (psgState[psg_NEW][i]==psgState[psg_PREV][i]) continue;
+    // if optimize=true, suppress writes to PSG voices that are not audible (volume=0 or R+L=0)
+    // ZSMKit has a feature that can benefit from having silent channels
+    // updated, so this is something that can be toggled off or on for export
+    if (optimize && i%4!=2 && (psgState[psg_NEW][(i&0x3c)+2]&0x3f)==0) continue; // vol
+    if (optimize && i%4!=2 && (psgState[psg_NEW][(i&0x3c)+2]&0xc0)==0) continue; // R+L
```

Comment:
Missing parentheses in `i%4`
Also, why not use `i&3`? `i%4` is slower.

Reply:
I have no good reason, & is better.  Thanks.